pkg/ssf_events/ssf_event.go

```diff
@@ -116,11 +119,13 @@ func EventStructFromEvent(eventUri string, eventSubject interface{}, claimsJson
 		return &event, nil
 	}
 
-	timestamp, err := strconv.ParseInt(subjectAttributes["timestamp"].(string), 10, 64)
-	if !ok || err != nil {
-		return nil, errors.New("unable to parse event subject")
+	floatTimestamp, ok := subjectAttributes["event_timestamp"].(float64)
```

Comment:
Why do you need to convert to float?

Reply:
Can you not use int64?

Reply:
The JSON parser by default parses every type of number to float64. So, we need to extract the float64 type first and then convert it to int64